frontend/public/locales/fr/apply.json

```diff
@@ -23,7 +23,7 @@
     "intro-text": "Merci d'utiliser la demande en ligne du Régime canadien de soins dentaires (RCSD) («\u00a0la demande en ligne\u00a0»). Nous avons établi conditions d'utilisation de ce service («\u00a0conditions d'utilisation\u00a0»). Veuillez les lire attentivement.",
     "terms-and-conditions-of-use": {
       "summary": "Conditions générales d'utilisation",
-      "online-application-legal-terms": "VEUILLEZ LIRE ATTENTIVEMENT. Les présentes conditions d'utilisation de la demande en ligne du Régime canadien de soins dentaires (RCSD) contiennent des renseignements sur vos droits et obligations juridiques et régissent votre utilisation de la demande en ligne. Les présentes conditions d'utilisation doivent être lues en parallèle avec celles du site Web Canada.ca. En cas de divergence, les présentes conditions d'utilisation ont préséance sur <canadaTermsConditions>celles du site Web Canada.ca</canadaTermsConditions>. En cas de divergence, les présentes conditions générales d'utilisation ont préséance sur celles du site Web Canada.ca.",
+      "online-application-legal-terms": "<strong>VEUILLEZ LIRE ATTENTIVEMENT</strong>. Les présentes conditions d'utilisation de la demande en ligne du Régime canadien de soins dentaires (RCSD) contiennent des renseignements sur vos droits et obligations juridiques et régissent votre utilisation de la demande en ligne. Les présentes conditions d'utilisation doivent être lues en parallèle avec celles du site Web Canada.ca. En cas de divergence, les présentes conditions d'utilisation ont préséance sur celles du site Web <canadaTermsConditions>Canada.ca</canadaTermsConditions>.",
```

Comment:
Period of `VEUILLEZ LIRE ATTENTIVEMENT` should be inside `strong` tags.

Reply:
`En cas de divergence, les présentes conditions générales d'utilisation ont préséance sur celles du site Web Canada.ca.` is still present in Figma unless I'm not looking at the right mockup.

Reply:
Bad notepad++ comparison, re-added missing line. Moved period inside tags